frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/print/elements/NodePrintElement.java

```diff
@@ -0,0 +1,45 @@
+
+package com.virtuslab.gitmachete.frontend.graph.print.elements;
+
+import com.intellij.vcs.log.graph.api.elements.GraphElement;
+import org.checkerframework.checker.index.qual.NonNegative;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import com.virtuslab.gitmachete.frontend.graph.GraphElementManager;
+
+public class NodePrintElement extends PrintElementWithGraphElement implements INodePrintElement {
```

Comment:
Maybe even replace `Node` with `Vertex` everywhere to make it super-unambiguous what you mean (if that's you mean indeed & unless it's gonna clash with some other naming we don't control etc.)

Reply:
Probably current structure clarifies things enough